DataFormats/PatCandidates/src/classes_def_other.xml

```diff
@@ -1,6 +1,9 @@
 <lcgdict>
  <selection>
 
+   <class name="std::vector<std::vector<float> >" />
+   <class name="edm::Wrapper<std::vector<std::vector<float> > >" />
```

Comment:
Yes, there are a few things that should be in the ECAL timing PR. I'll move
them.

On Fri, Sep 9, 2016 at 2:53 PM, Slava Krutelyov <notifications@github.com>
wrote:

> In DataFormats/PatCandidates/src/classes_def_other.xml
> <https://github.com/cms-sw/cmssw/pull/15710#discussion_r78238891>:
>
> > @@ -1,6 +1,9 @@
> >  <lcgdict>
> >   <selection>
> >
> > +   <class name="std::vector<std::vector<float> >" />
> > +   <class name="edm::Wrapper<std::vector<std::vector<float> > >" />
>
> The dictionary for this STL-based type is already available in
> https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/
> DataFormats/StdDictionaries/src/classes_def_vector.xml#L32
>
> please remove it from this file.
>
> the pair string-vectorFloat below is also misplaced, but I suppose it's
> not for you
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/15710/files/975fed1116b68135791ddd1df9433953c7fbb37b#r78238891>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABBMOTH2UWJQbBdm2hbPkE3wmSqZj6aNks5qobk8gaJpZM4Jz-C->
> .
>